setup.py

```diff
@@ -0,0 +1,69 @@
+#!/bin/env python
+
+import os
+import logging
+import notify2
+from time import perf_counter
+from dotenv import load_dotenv
+from fetch_mail import LPForum
+from send_mail import SendToLPForum
+from random_message import GenerateMessage
+
+def Notify(summary, data, urgency, timeout=12000):
+    """ Create a notification pop-up
+    to aware the user about task status
+    """
+
+    notify2.init("AutoReplyToLP")
```

Comment:
It might be better to do init outside of function calls. This init line will be called every time this function is called.

Normally init stuff only happens one time on startup.